inc/thorvg.h

```diff
@@ -1286,7 +1286,7 @@ class TVG_API Picture final : public Paint
      * when the @p copy has @c false. This means that loading the same data again will not result in duplicate operations
      * for the sharable @p data. Instead, ThorVG will reuse the previously loaded picture data.
      *
-     * @param[in] data A pointer to a memory location where the content of the picture file is stored.
+     * @param[in] data A pointer to a memory location where the content of the picture file is stored (null-terminated for non-binary data if @p copy is @c false).
```

Comment:
I think we can clarify the sentence more. `A null-terminated string is expected for non-binary data if @p copy is @c false`